packages/nextjs/components/ui/button.tsx

```diff
@@ -4,21 +4,21 @@ import { type VariantProps, cva } from "class-variance-authority";
 import { cn } from "~~/lib/utils";
 
 const buttonVariants = cva(
-  "inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:pointer-events-none disabled:opacity-50",
+  "inline-flex items-center justify-center whitespace-nowrap rounded-full text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-0 focus-visible:ring-ring disabled:pointer-events-none disabled:opacity-50",
   {
     variants: {
       variant: {
         default: "bg-primary text-primary-foreground shadow hover:bg-primary/90",
         destructive: "bg-destructive text-destructive-foreground shadow-sm hover:bg-destructive/90",
         outline: "border border-input bg-background shadow-sm hover:bg-accent hover:text-accent-foreground",
         secondary: "bg-secondary text-secondary-foreground shadow-sm hover:bg-secondary/80",
-        ghost: "hover:bg-accent hover:text-accent-foreground",
+        ghost: "hover:bg-accent",
```

Comment:
Color is not changed intentionally. Scroll to ghost in docs https://ui.shadcn.com/docs/components/button, color is very organic, so when we'll update our theme I think it will be fixed. 

And I also think that when possible we don't need change colors in `ui` folder components. Again it should work great everywhere after configuring theme